src/util/getOptions.js

```diff
@@ -20,6 +20,6 @@ export default function getOptions(that, resource, element, emptyOption) {
   }
 
   return res.sort((a, b) => (a.label.toLowerCase() < b.label.toLowerCase() ? -1 :
-                             a.label.toLowerCase() > b.label.toLowerCase() ? 1 :
-                             0));
+    a.label.toLowerCase() > b.label.toLowerCase() ? 1 :
```

Comment:
I consider this indentation significantly less readable than what we had before, but _shrug_ I guess if that's what ESLint wants, then that's what matters.

Reply:
I agree with all words in that sentence.

Reply:
I love that this process is both fruitful _and_ funny :-)